src/components/course/CourseContextProvider.jsx

```diff
@@ -1,20 +1,21 @@
 import React, { createContext, useReducer } from 'react';
 import PropTypes from 'prop-types';
+import { SET_COURSE_RUN, SET_COURSE, SET_ENROLLMENTS, SET_ENTITLEMENTS, SET_IS_COURSE_IN_CATALOG } from './data/constants';
```

Comment:
Seemed better to constantize(ify?) these before we decide to refactore a name at some point and change it in a bunch of locations.